.travis.yml

```diff
@@ -86,7 +121,7 @@ jobs:
         - MB_PYTHON_VERSION=3.9
         - PLAT=aarch64
         - CYTHON_BUILD_DEP="Cython"
-        - DOCKER_TEST_IMAGE=multibuild/xenial_arm64v8
```

Comment:
These two don't need the update to focal for testing. The reason to use different docker images for building and testing is to test on old Ubuntu versions. That couldn't be done for 3.10 because, IIRC, deadsnakes didn't provide 3.10 for xenial. @mattip Do you recall the reasons? I note that deadsnakes was recently updated.